test/azure/generated/AzureResource/models/mappers.ts

```diff
@@ -14,205 +14,179 @@ export const CloudError = CloudErrorMapper;
 export const BaseResource = BaseResourceMapper;
 
 export const ErrorModel = {
-  required: false,
-  serializedName: 'Error',
+  serializedName: "Error",
```

Comment:
does the linter prefer double quotes? I seem to recall our ms-rest-js and autorest.typescript linters having contradictory settings.

Reply:
That sounds familiar, but I'm just going to go with whatever settings we use in the ms-rest-js repository. If we start getting linter errors in autorest.typescript, then we should update the autorest.typescript linter rules to be the same as ms-rest-js.